app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,101 @@
  *= require_tree .
  *= require_self
  */
+#new_user {
+  margin: 50px auto;
+  width: 280px;
+  padding: 33px 25px 29px;
+  background: #AA1925;
+  border-bottom: 1px solid #c4c4c4;
+  border-radius: 5px;
+  box-shadow: 0 1px 5px rgba(0, 0, 0, 0.25);
+}
```

Comment:
In general when styling it's best to add a class to the element and then use that to target styling — classes for styling, IDs (and sometimes other things) for Javascript targetting.